modules/series-service-impl/src/main/java/org/opencastproject/series/impl/SeriesServiceImpl.java

```diff
@@ -256,19 +189,22 @@ public DublinCoreCatalog updateSeries(DublinCoreCatalog dc) throws SeriesExcepti
         }
 
         logger.debug("Updating series {}", id);
-        index.updateIndex(dublinCore);
+        // ??? Is this even necessary? This function is about updating DC after all???
         try {
           final AccessControlList acl = persistence.getAccessControlList(id);
           if (acl != null) {
-            index.updateSecurityPolicy(id, acl);
+            updateSeriesAclInIndex(id, elasticsearchIndex, acl);
           }
         } catch (NotFoundException ignore) {
           // Ignore not found since this is the first indexing
         }
-        // Make sure store to persistence comes after index, return value can be null
-        DublinCoreCatalog updated = persistence.storeSeries(dublinCore);
+
         // update API index
         updateSeriesMetadataInIndex(id, elasticsearchIndex, dublinCore);
```

Comment:
Does this update the series metadata in the external index and the admin-ui index or are they the same at this point in this version of OC?